src/Queue/Concerns/UsesCommand.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LaraStrict\Queue\Concerns;
+
+use Illuminate\Console\Command;
+
+trait UsesCommand
+{
+    private ?Command $command = null;
+
+    public function setCommand(?Command $command): void
```

Comment:
Chceš mít možnost resetovat command? pokud ne odebral bych otazník

Reply:
dobrý point, určitě bude zbytečné